WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -124,9 +124,9 @@ extension WooConstants {
         case addPaymentMethodWCShip = "https://wordpress.com/me/purchases/add-payment-method"
 
 #if DEBUG
-        case quickOrderPrototypeFeedback = "https://automattic.survey.fm/woo-app-quick-order-testing"
+        case simplePaymentsPrototypeFeedback = "https://automattic.survey.fm/woo-app-simple-payments-testing"
 #else
-        case quickOrderPrototypeFeedback = "https://automattic.survey.fm/woo-app-quick-order-production"
+        case simplePaymentsPrototypeFeedback = "https://automattic.survey.fm/woo-app-simple-payments-production"
```

Comment:
It looks like these two surveys don't exist yet, so I'm getting a 404 error when I tap the "Give Feedback" button on the banner. Who has access to set these up?

Reply:
Wow, I totally missed that! Thanks for spotting it!

I have renamed the survey URLs to their previous value so we don't end up with two different surveys. (The one being released in 7.9 and this one 8.0+)

https://user-images.githubusercontent.com/562080/140949439-9461767f-6253-434f-a298-8d9d5b9fc30d.mov